WireUI/Sources/WireIndividualToTeamMigrationUI/Analytics/AnalyticsEvents.swift

```diff
@@ -0,0 +1,120 @@
+//
+// Wire
+// Copyright (C) 2024 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import WireAnalytics
+
+extension AnalyticsEvent.User {
+
+    /// Count of user reaching for each modal step (Step 1 through 3)
+    ///
+    /// Segmentation: app_name; app_version
+
+    static func personalTeamCreationFlowStarted(
+        step: IndividualToTeamMigrationViewController.Step
+    ) -> AnalyticsEvent! {
```

Comment:
Because this event is not supposed to be created for the `.completion` value of `Step`.
Which is the most preferred alternative?
1. Making the return value `AnalyticsEvent?` and adding a check to the calling code.
2. `fatalError()` or similar in the `switch`.
3. Returning an empty event.
4. Keep it
5. Anything I cannot think of right now?

Reply:
The code is now different, the force-unwrap is gone.